Freshli/Util/DateTimeHelper.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace Freshli.Util {
+  public static class DateTimeHelper {
+    public static DateTime ConvertToEndOfDay(DateTime date) {
```

Comment:
What if we set this up as an extension instead?

Something like `DateTimeExtensions.cs` would contain:

```
public static DateTime ToEndOfDay(this DateTime date)
{
    return date.Date.AddDays(1).AddTicks(-1);
}
```

You'd then be able to call it like so:

```
DateTime.Today.ToEndOfDay()
```

or:
```
var inputDateEoD = inputDate.ToEndOfDay();
```

Reply:
I like that feature! I'll check it in shortly. Thanks for the suggestion!